File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        for i, n in enumerate(nums):
            v = target - n
            
            if v in nums[i+1:]: return [nums.index(n), nums[i+1:].index(v) + (i+1)]

File: 0001-two-sum/0001-two-sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    const jsMap = new Object();
    for(let i = 0; i < nums.length; i++){
        n = nums[i]
        if(target - n in jsMap){
            return [jsMap[target - n], i]
           };
        jsMap[n] = i
        }
};